cpp/src/lib/rora_proxy_client.cc

```diff
@@ -221,20 +229,38 @@ int RoraProxy_client::_short_path_many(
   return result;
 }
 
+void _process(std::vector<object_info> &object_infos,
+              const std::string &namespace_) {
+
+  ALBA_LOG(DEBUG, "_process : " << object_infos.size());
+  for (auto &object_info : object_infos) {
+    using alba::stuff::operator<<;
+    // ALBA_LOG(DEBUG, "_procesing object_info:" << object_info);
+
+    const std::string &object_name = std::get<0>(object_info);
+    // auto& future = std::get<1>(object_info);
```

Comment:
Should check here whether future has the expected value. If not it should not be added to the manifest cache.